loop_lab.js

```diff
@@ -0,0 +1,173 @@
+// // 1. Write a `while` loop and a `for` loop that takes the variable
```

Comment:
Good demonstration of for and while loops overall. Other than minor mistake with logic for one problem, everything looks fine. Good job and keep it up!